shop4All-backend/src/main/java/com/example/shop4All_backend/services/JwtService.java

```diff
@@ -49,11 +47,12 @@ public JwtResponse createJwtToken(JwtRequest jwtRequest) throws AuthenticationEx
             user = userOptional.get();
         }
         else{
+            System.out.println("---------");
```

Comment:
In a backend application, you will be using log (Logger) instead of prints. Take this comment into consideration in the future PR. You can also refer to this [link](https://www.baeldung.com/java-system-out-println-vs-loggers#:~:text=With%20the%20use%20of%20Loggers,timestamp%2C%20method%20name%2C%20etc.).